Scripts/BuildPhases/LintAppLocalizedStringsUsage.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+SCRIPT_DIR="$(dirname "${BASH_SOURCE[0]}")"
+SCRIPT_SRC="${SCRIPT_DIR}/LintAppLocalizedStringsUsage.swift"
+
+LINTER_BUILD_DIR="${BUILD_DIR:-${TMPDIR}}"
+LINTER_EXEC="${LINTER_BUILD_DIR}/$(basename "${SCRIPT_SRC}" .swift)"
+
+if [ ! -x "${LINTER_EXEC}" ] || ! (shasum -c "${LINTER_EXEC}.shasum" >/dev/null 2>/dev/null); then
+  echo "Pre-compiling linter script to ${LINTER_EXEC}..."
+  swiftc -sdk "$(xcrun --sdk macosx --show-sdk-path)" "${SCRIPT_SRC}" -o "${LINTER_EXEC}"
```

Comment:
Do you think the shell script should exit with error when failing to compile the Swift script? I wouldn't be surprised if Swift 6 or something breaks source code compatibility.

Reply:
That's a good point. The shell script is called by the Script Build Phase using `/bin/bash -eu` And the `-e` flag would make the script exit on first error.

But I'm not sure that flag carries over when the script from the Script Build Phase itself calls this `LintAppLocalizedStringsUsage.sh` one. Maybe I should add `-eu` to the shebang of this file as well, just to be sure.

Reply:
Addressed in 2ea977a2ee